core/drivers/crypto/crypto_api/include/drvcrypt_acipher.h

```diff
@@ -149,10 +149,11 @@ struct drvcrypt_ecc_ed {
  */
 struct drvcrypt_ecc {
 	/* Allocates the ECC keypair */
-	TEE_Result (*alloc_keypair)(struct ecc_keypair *key, size_t size_bits);
+	TEE_Result (*alloc_keypair)(struct ecc_keypair *key, size_t size_bits,
+				    uint32_t type);
```

Comment:
We may want to swap the order of `size_bits` and `type` to match `drvcrypt_asym_alloc_ecc_keypair()`